pact/Pact/Core/Trans/MPFR.hs

```diff
@@ -157,6 +157,17 @@ foreign import ccall "mpfr_exp"
 foreign import ccall "mpfr_sqrt"
   c'mpfr_sqrt :: Mpfr_t -> Mpfr_t -> CInt -> IO ()
 
+foreign import ccall "mpfr_number_p"
```

Comment:
`mpfr_number_p` just for review's sake: checks whether the operand returned is a number, and not `NaN` or `Inf`.